.gitignore

```diff
@@ -77,3 +77,4 @@ shake.yaml.lock
 
 # ignore hie.yaml's for testdata 
 test/**/*.yaml
+/hie.yaml
```

Comment:
Do we want to gitignore this actually? At some point should we be adding one into the hie repo itself? cc @alanz @fendor

Reply:
If we have one in repo should it use stack or cabal?

Reply:
I am in general unsure how to handle `hie.yaml` files. Should they be ignored because we can expect everyone to write one themselves? Should we add two hie.yamls one for cabal and stack and users rename them as needed? Or default to cabal or stack?
none of these options seem desirable, at the moment.

Reply:
I think we should not commit hie.yaml directly.  We can perhaps provide templates, as @fendor suggests.  Different people build different ways, we should be agnostic to that.

Reply:
@alanz so leave it?